### PR TITLE
Add other integral drop_chunks() variants

### PR DESCRIPTION
**Pull request by RobAtticus**
Previously drop_chunks() only took INTEGER, which prevented it
from being called with BIGINT values, e.g. for nanoseconds.

Addresses https://github.com/timescale/timescaledb/issues/260

### CODE REVIEW

File: sql/ddl_api.sql [hunk 1]
```diff
@@ -224,6 +224,34 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION drop_chunks(
```
Comment (cevian) [inline sql/ddl_api.sql]: I don't think you need this version. Aren't INTEGERS auto-casted to BIGINT on function calls?

**Comment by RobAtticus** [comment]
Removed other variants leaving just BIGINT (and updated commit message).

File: test/sql/drop_chunks.sql [hunk 2]
```diff
@@ -70,6 +71,20 @@ WHERE h.schema_name = 'public' AND (h.table_name = 'drop_chunk_test1' OR h.table
 
 \dt "_timescaledb_internal".*
 
+-- 2,147,483,647 is the largest int so this tests that BIGINTs work
+SELECT drop_chunks(2147483648, 'drop_chunk_test3');
```
Comment (cevian) [inline test/sql/drop_chunks.sql]: Shouldn't we actually insert some data into `drop_chunk_test3` before dropping chunks so we can test that it actually does something?

Comment (RobAtticus) [reply test/sql/drop_chunks.sql]: Bah sorry forgot that when I redid.

File: test/expected/drop_chunks.out [hunk 3]
```diff
@@ -1,5 +1,6 @@
 CREATE TABLE PUBLIC.drop_chunk_test1(time bigint, temp float8, device_id text);
 CREATE TABLE PUBLIC.drop_chunk_test2(time bigint, temp float8, device_id text);
+CREATE TABLE PUBLIC.drop_chunk_test3(time bigint, temp float8, device_id text);
```
Comment (cevian) [inline test/expected/drop_chunks.out]: drop_chunk_test3 was never made into a hypertable with create_hypertable. Come to think of it, that should be an error in drop chunks. Can you change?

Comment (RobAtticus) [reply test/expected/drop_chunks.out]: Done

**Pull request merged**